internal/admin/dashboard/templates/page-overview.html

```
@@ -0,0 +1,217 @@
+{{define "dashboard-page-overview"}}
+<!-- Overview Page -->
+<template x-if="page==='overview'">
+<div>
+    <div class="page-header">
+        <h2>Usage Overview</h2>
+        <div class="page-header-controls">
+            <div class="interval-picker">
+                <button class="interval-btn" :class="{active: interval==='daily'}" @click="setInterval('daily')">Daily</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)